packages/ui/layouts/WizardLayout.tsx

```diff
@@ -3,15 +3,16 @@ import { useRouter } from "next/router";
 import React, { useEffect, useState } from "react";
 import { Toaster } from "react-hot-toast";
 
-import { StepCard, Steps } from "@calcom/ui";
+import { StepCard, Steps, Button } from "@calcom/ui";
 
-export default function WizardLayout({
+export function WizardLayout({
   children,
   maxSteps = 2,
   currentStep = 0,
+  isOptionalCallback,
 }: {
   children: React.ReactNode;
-} & { maxSteps?: number; currentStep?: number }) {
+} & { maxSteps?: number; currentStep?: number; isOptionalCallback?: () => void }) {
```

Comment:
Added the option to include a button to skip the step if it is marked as optional, providing a callback to know what to do once it is clicked